package.json

```diff
@@ -153,7 +153,7 @@
     "src"
   ],
   "scripts": {
-    "build": "gulp build && untracked > .nowignore",
+    "build": "gulp build && untracked > .vercelignore",
```

Comment:
@Kikobeats The only thing I'm not sure about is making sure `gulp build` outputs to the `public` directory